src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -40,6 +41,7 @@
 @Service
 public class AdminService {
 
+    private static final String AES_KEY_LOG_DESCRIPTION = "AES Key";
```

Comment:
Really minor, I tend to prefer gathering constants together for clarity.
Would it be possible to have this definition next to `AES_KEY_FILENAME_EXTENSION` definition ?

Reply:
done locally